### ISSUE TITLE
Changing graphic options in native Linux invalidates textures

### ISSUE THREAD

**Issue by RattleSN4K3** [UltraStar-Deluxe/USDX#55]
The issue is known for Windows/Mac systems, but it seems like it is the case for Linux as well (at least running in Lubuntu [in a VM]).

> Unit: UScreenOptionsGraphics  
Method: TScreenOptionsGraphics.ParseInput  
```
            // FIXME: changing the video mode does not work this way in windows
            // and MacOSX as all textures will be invalidated through this.
            // See the ALT+TAB code too.
            {$IF Defined(Linux) or Defined(FreeBSD)}
            Reinitialize3D();
            {$IFEND}  
```

![lubuntu-2016-07-15-20-20-12](https://cloud.githubusercontent.com/assets/6833006/16884449/c5c1f4e6-4ac9-11e6-8256-694bb5ba19a0.png)

**Comment by basisbit** [comment]
oh yes, this was changed with SDL2. please comment out that line and see if this fixed the issue

**Comment by basisbit** [comment]
apparently also depends on used graphics drivers.

**Comment by RattleSN4K3** [comment]
> oh yes, this was changed with SDL2. please comment out that line and see if this fixed the issue
Textures don't invalidate on commenting out `Reinitialize3D();`.

>apparently also depends on used graphics drivers.
3D accelerated. Emulated though.

**Comment by RattleSN4K3** [comment]
Fixed by #60.
